queue_block.py

```diff
@@ -75,9 +78,13 @@ def stop(self):
             self._emit_job.cancel()
         super().stop()
 
-    def process_signals(self, signals):
-        self.logger.debug("Processing {} signals".format(len(signals)))
-        self.for_each_group(self._push_group, signals)
+    def process_signals(self, signals, input_id="__default_terminal_value"):
```

Comment:
This string shouldn't be hardcoded, we should use `nio.block.terminals.DEFAULT_TERMINAL` or something like that